event-statistics/src/main/avro/fight.avsc

```diff
@@ -6,8 +6,8 @@
   "fields": [
     {
       "name": "id",
-      "type": "long",
-      "default": -1
+      "type": "string",
+      "default": ""
```

Comment:
shouldn't we use `null`?

Reply:
Actually I'll do one better. It wasn't until just now that I understood how `null` works in Avro :) I'll push a new commit in a minute or 2.

Reply:
@cescoffier I pushed a new commit.